scripts/create-topics.sh

```diff
@@ -47,6 +47,10 @@ done
 vuln_analysis_topics=(
   "dtrack.vuln-analysis.component"
   "dtrack.vuln-analysis.result"
+  "dtrack.vulnerability.mirror.osv"
+  "dtrack.vulnerability.mirror.nvd"
```

Comment:
Hmmm, considering the NVD mirroring can't be split into multiple sub-tasks like OSV can, and we really only want the NVD mirroring to happen once at any given time, would it make sense to only have one partition for `dtrack.vulnerability.mirror.nvd`? 🤔 At the moment it will have 3.